Dockerfile

```diff
@@ -14,17 +14,21 @@ ENV PYTHON_VERSION_36 3.6.12
 ENV PYTHON_VERSION_37 3.7.9
 ENV PYTHON_VERSION_38 3.8.6
 ENV PYTHON_VERSION_39 3.9.1
+ENV PYTHON_VERSION_310 3.10.0b4
 ENV PYPY_VERSION_35 pypy3.5-7.0.0
-# Note: 4.7.12.1 drastically increases memory usage
-ENV CONDA_VERSION 4.6.14
+# Latest miniconda version is Miniconda3-py39_4.10.1, but it's not available for x86_64
+ENV CONDA_PYTHON_VERSION py39
+ENV CONDA_VERSION 4.9.2
 LABEL python.version_27=$PYTHON_VERSION_27
 LABEL python.version_35=$PYTHON_VERSION_35
 LABEL python.version_36=$PYTHON_VERSION_36
 LABEL python.version_37=$PYTHON_VERSION_37
 LABEL python.version_38=$PYTHON_VERSION_38
 LABEL python.version_39=$PYTHON_VERSION_39
+LABEL python.version_310=$PYTHON_VERSION_310
 LABEL pypy.version_35=$PYPY_VERSION_35
 LABEL conda.version=$CONDA_VERSION
+LABEL conda.python_version=$CONDA_PYTHON_VERSION
 
 # System dependencies
 RUN apt-get -y update
```

Comment:
Unrelated to this PR, but https://www.fromlatest.io/ says:

> Use of apt-get update should be paired with rm -rf /var/lib/apt/lists/* in the same layer.

> All instances of apt-get update should have the apt-get install commands on the same line to reduce image size.

> Consider using a --no-install-recommends when apt-get installing packages. This will result in a smaller image size. For more information, see this blog post

Reply:
Yeah, our images are not super optimized, unfortunately. See #60

Reply:
Thanks! Let's continue the conversation there